start_notebook.sh

```diff
@@ -76,12 +77,36 @@ jupyter ${JUPYTER_TYPE} --no-browser --port=$PORT --ip=\$JUP_HOST --notebook-dir
 EOF
 chmod +x $INNER
 
-if [[ "$PARTITION" == "dali" ]]; then
-  export XENON_CONFIG=/dali/lgrandi/xenonnt/xenon.config
-elif [[ "$PARTITION" == "lgrandi" || "$PARTITION" == "build" || "$PARTITION" == "caslake" ]]; then
-  export XENON_CONFIG=/project/lgrandi/xenonnt/xenon.config
+# Set default XENON_CONFIG path based on PARTITION
+case "$PARTITION" in
+  "dali")
+    DEFAULT_CONFIG="/dali/lgrandi/xenonnt/xenon.config"
+    ;;
+  "lgrandi" | "build" | "caslake")
+    DEFAULT_CONFIG="/project/lgrandi/xenonnt/xenon.config"
+    ;;
+  *)
```

Comment:
This isn't working on midway2. Please add the midway2 partitions.

Reply:
Many thanks for spotting it! I have added the midway2 partitions